rotkehlchen/constants/assets.py

```diff
@@ -266,6 +266,7 @@
 A_PSP = Asset('eip155:1/erc20:0xcAfE001067cDEF266AfB7Eb5A286dCFD277f3dE5')
 A_SDL = Asset('eip155:1/erc20:0xf1Dc500FdE233A4055e25e5BbF516372BC4F6871')
 A_VCOW = Asset('eip155:1/erc20:0xD057B63f5E69CF1B929b356b579Cba08D7688048')
+A_VCOW_GNOSIS = Asset('eip155:100/erc20:0xc20C9C13E853fc64d054b73fF21d3636B2d97eaB')
 A_FPIS = Asset('eip155:1/erc20:0xc2544A32872A91F4A553b404C6950e89De901fdb')
```

Comment:
by comparing with others like `A_POLYGON_POS_MATIC` and `A_OPTIMISM_ETH` we seem to have the chain before the asset name so it should be `A_GNOSIS_VCOW`